code/game/objects/items/devices/radio/radio.dm

```diff
@@ -209,6 +210,7 @@
 		spans = list(M.speech_span)
 	if(!language)
 		language = M.get_selected_language()
+			SEND_SIGNAL(M, COMSIG_MOVABLE_RADIO_TALK_INTO, src, message, channel, spans, language, direct) //SKYRAT EDIT ADDITION - GUNPOINT
```

Comment:
```suggestion
		SEND_SIGNAL(M, COMSIG_MOVABLE_RADIO_TALK_INTO, src, message, channel, spans, language, direct) //BUNGALOW EDIT ADDITION - GUNPOINT
```
First: Indentation.
Second: You ain't SkyRay downstream.

Reply:
Yeah, thx for pointing out the indent

We ain't skyray downstream, but I am leaving "skyrat edit addition" for purpose to point out where module comes from, especially since it uses skyrat signals